src/components/molecules/LiveSchedule/LiveEvent.tsx

```diff
@@ -10,17 +10,21 @@ import { WithId } from "utils/id";
 import { useUser } from "hooks/useUser";
 
 interface LiveEventProps {
-  venue: WithId<Venue>;
-  event: VenueEvent;
+  venue?: WithId<Venue>;
+  event?: VenueEvent;
 }
 
 export const LiveEvent: FC<LiveEventProps> = ({ venue, event }) => {
   const { user, profile } = useUser();
 
   const enterLiveEvent = useCallback(() => {
-    openEventRoomWithCounting({ user, profile, venue, event });
+    if (venue && event) {
+      openEventRoomWithCounting({ user, profile, venue, event });
+    }
   }, [event, profile, user, venue]);
 
+  if (!event) return <></>;
```

Comment:
Returning `null` is the correct pattern here, fragment is only used to group things when you don't want an actual div to be emitted.

- https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

```suggestion
  if (!event) return null;
```